pkg/providers/amifamily/windows.go

```diff
@@ -47,7 +47,7 @@ type Windows struct {
 }
 
 func (w Windows) DescribeImageQuery(ctx context.Context, ssmProvider ssm.Provider, k8sVersion string, amiVersion string) (DescribeImageQuery, error) {
-	imageID, err := ssmProvider.Get(ctx, fmt.Sprintf("/aws/service/ami-windows-latest/Windows_Server-%s-English-%s-EKS_Optimized-%s/image_id", w.Version, v1.WindowsCore, k8sVersion))
+	imageID, err := ssmProvider.Get(ctx, w.SSMParameters(k8sVersion, amiVersion)[0])
```

Comment:
You could probably do this without leaking the fact that you only have a single parameter -- realistically this function shouldn't know that